.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Build and publish
+
+# Run on PR requests. And on master itself.
+on:
+  push:
+    branches:
+      - main  # just build the sdist skip release
+    tags:
+      - "*"
+  pull_request:  # also build on PRs touching some files
+    paths:
+      - ".github/workflows/release.yml"
+      - "MANIFEST.in"
+      - "setup.cfg"
+      - "setup.py"
```

Comment:
Deze zijn weg? En misschien de pyproject.toml toevoegen (maar dat hangt er van af wat je doel qua uploaden naar github releases is).